src/en/articles/landmarks/index.njk

```diff
@@ -1,136 +1,204 @@
 ---
 title: "Landmarks ARIA"
```

Comment:
It may be a good idea to make this title more descriptive, like the french version ('Structurer une page avec les régions (landmarks) ARIA')